parachain/pallets/ethereum-beacon-client/tests/fixtures/minimal_initial_sync.json

```diff
@@ -45,10 +45,10 @@
     },
     "current_sync_committee_branch": [
         "0x92df9cdb8a742500dbf7afd3a7cce35805f818a3acbee8a26b7d6beff7d2c554",
-        "0x3386e0343ff03c4a148563dd4cbb5a7ad6947087ea6f6e7969a23ff52c535a11",
-        "0xdc064afa8fe4539c0b6bfbbf8090bad226fa7c884c2ec433298a4171bee5fc4d",
-        "0x3d5d83ed07ab8f6623824c1f1db0ac955636b96d07c24fce50a4c456b2264f55",
-        "0x791843f852bba1c691eb0f060da8c9700dd6b40675f870a0d305b79af79ea86d"
+        "0x058baa5628d6156e55ab99da54244be4a071978528f2eb3b19a4f4d7ab36f870",
+        "0x5f89984c1068b616e99589e161d2bb73b92c68b3422ef309ace434894b4503ae",
+        "0xd33a17a3903ceac967c0afc2be32962dd69f5836e7674b4c30b2c68116720b2c",
+        "0x0d0607530d6ffd3dfffafee157c34db1430cd7a1f29dea854769cf5c45aed99d"
     ],
     "validators_root": "0x270d43e74ce340de4bca2b1936beca0f4f5408d9e78aec4850920baf659d5b69"
 }
```

Comment:
ditto

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/733/commits/e5ea69c139c00ff46d543b0363a6297299a371c7.